storage/innobase/handler/ha_innodb.cc

```diff
@@ -3909,6 +3909,8 @@ innobase_fix_tablespaces_empty_uuid()
 		return(false);
 	}
 
+	log_enable_encryption_if_set();
```

Comment:
Since encryption of redo log is enabled from here, we can remove  log_eneable_encryption_if_set() from srv_master_thread tasks.

Reply:
Fixed - this also uncovered a missing rotation call, which caused crashes.